packages/manager/apps/pci-private-registry/src/components/guides-header/guides-header-item.component.tsx

```diff
@@ -0,0 +1,48 @@
+import { OsdsIcon, OsdsLink, OsdsMenuItem, OsdsButton, OsdsMenuGroup } from '@ovhcloud/ods-components/react';
+import { ODS_BUTTON_VARIANT, ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { OdsHTMLAnchorElementTarget } from '@ovhcloud/ods-common-core';
+import { Guide } from './interface';
+
+interface GuidesHeaderItemProps {
+  guide: Guide;
+  href: string;
+  label: string;
+  tracking?: string;
+  onClick?: (guide: Guide) => void;
+}
+
+export function GuidesHeaderItem({
```

Comment:
Thought: it looks like there are some formatting issue like `<OsdsMenuGroup>` and `<OsdsMenuItem>` being at the same level and some unnecessary empty lines.